express-mongoose_Rohit_Yadav/README.md

```diff
@@ -0,0 +1,25 @@
+#   Hotel Management Web App
+
+ 
+ 
+
+ 
+1. **Node.js (version 14 or later):** Download and install it from [https://nodejs.org/en](https://nodejs.org/en).
+2. **npm (version 5.6 or later):** This usually comes bundled with Node.js. Verify by running `npm -v` in your terminal.
+
+**1. Clone the Repository and Install Dependencies:**
+
+```bash
+git clone  https://github.com/Rohitrky2021/CRUD_Hotel_App
+cd cd Hotel-Management
+npm install
+```
+
+**2. Run the Development Server:**
+Run the following command to launch the development server:
+
+```Bash
+npm run dev
+```
+
```

Comment:
can you also add the steps needed, after running npm run dev and how to run with keploy ?

Reply:
I was actually having a problem setting up WSL in my Windows, that's why. Now I have updated it properly.